schema/deploy/warehouse/identifier/triggers/barcode-distance-check.sql

```diff
@@ -44,8 +46,8 @@ create or replace function warehouse.identifier_barcode_distance_check() returns
                         TG_OP, NEW.barcode),
 
                 detail = format(
-                    'Barcode %L is closer than %s substitutions away from existing barcodes %s',
-                        NEW.barcode, minimum_distance, conflicting_barcodes),
+                    'Barcode %L is closer than %s substitutions away from an existing barcode',
+                        NEW.barcode, minimum_distance),
```

Comment:
It could still be useful to include first_conflict here as an example of what caused the failure.

Reply:
Added back in fixup commit.